### PR TITLE
add Library/Formula/mytget.rb

### PR DESCRIPTION
**Pull request by lytsing**
Mytget is a muti-thread downloader accelerator for GNU/Linux.

### CODE REVIEW

File: Library/Formula/mytget.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
```
Comment (dunn) [inline Library/Formula/mytget.rb]: Remove all the comments, please!

File: Library/Formula/mytget.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! It's enough to just replace
+    # "false" with the main program this formula installs, but it'd be nice if you
+    # were more thorough. Run the test with `brew test mytget`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
+    system "false"
```
Comment (dunn) [inline Library/Formula/mytget.rb]: We'll need a test; something that checks basic functionality.

File: Library/Formula/mytget.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (bfontaine) [inline Library/Formula/mytget.rb]: Please remove this empty line.

File: Library/Formula/mytget.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  desc "Mytget is a muti-thread downloader accelerator for GNU/Linux. "
+  homepage "https://github.com/lytsing/mytget"
+  url "https://github.com/lytsing/mytget/archive/v1.1.2.tar.gz"
+  version "1.1.2"
+  sha256 "ac369d9c8ef99933973d108c9a1f5774cfcc9ddcc33267a619bffaa885f2824e"
+
+  depends_on "cmake" => :build
+
+  def install
+
```
Comment (bfontaine) [inline Library/Formula/mytget.rb]: Please remove this empty line.

File: Library/Formula/mytget.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system bin/"mytget", "https://google.com"
+  end
+end
+
```
Comment (bfontaine) [inline Library/Formula/mytget.rb]: Please remove this empty line.

File: Library/Formula/mytget.rb [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+class Mytget < Formula
+  desc "Mytget is a muti-thread downloader accelerator for GNU/Linux. "
+  homepage "https://github.com/lytsing/mytget"
+  url "https://github.com/lytsing/mytget/archive/v1.1.2.tar.gz"
+  version "1.1.2"
```
Comment (bfontaine) [inline Library/Formula/mytget.rb]: You can remove this line.

File: Library/Formula/mytget.rb [hunk 7]
```diff
@@ -0,0 +1,21 @@
+
+class Mytget < Formula
+  desc "Mytget is a muti-thread downloader accelerator for GNU/Linux. "
```
Comment (bfontaine) [inline Library/Formula/mytget.rb]: Please remove the trailing space.

**Comment by bfontaine** [comment]
The audit is failing:

```
 * Stable: version 1.1.2 is redundant with version scanned from URL
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
 * object files were linked against system openssl
These object files were linked against the deprecated system OpenSSL.
Adding `depends_on "openssl"` to the formula may help.
  /usr/local/Cellar/mytget/1.1.2/bin/mytget
```

**Comment by mikemcquaid** [comment]
I'm afraid this project is too niche for us currently. I suggest creating your own tap. Sorry!

**Pull request closed without merging**